### PR TITLE
Release notes for 1.16.0

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: content/release-notes/1.16.0.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+{{</changes>}}
+
+{{<fixes>}}
+* Fixed a bug that could cause `LocalImageName` and `LocalRegistryImagePullSecret` to panic.
+* Snapshots page will show a message when connection to the API is lost.
+* Fixed a bug that would cause orphaned pods to be left on the system when jobs are deleted.
+* Fixed a bug that could cause a mismatch between TLS cert and key when using `TLSCert` and `TLSKey` template functions
+* Added scrolling on airgap upload progress page.
+* Fixed a bug that allowed installing application with invalid YAML.
+* Fixed a bug that could cause operator to not bee deployed, resulting in applications being stuck in `Deploying` state indefinitely.
```
Comment (dexhorthy) [inline content/release-notes/1.16.0.md:32]:
```suggestion
* Fixed a bug that could cause operator to not be deployed, resulting in applications being stuck in `Deploying` state indefinitely.
```

File: content/release-notes/1.16.0.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+date: 2020-06-01T08:00:00-07:00
+linktitle: "1.16.0"
+title: "1.16.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+* Support for HTTP_PROXY and NO_PROXY environment variables.
```
Comment (austinchambers) [inline content/release-notes/1.16.0.md:9]: Where are these environment variables supported? Is this for embedded cluster installations? 

If so: 
```suggestion
* Support for HTTP_PROXY and NO_PROXY environment variables in embedded cluster installations.
```

Comment (laverya) [reply content/release-notes/1.16.0.md:9]: unless it also works in existing cluster installs

File: content/release-notes/1.16.0.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* CronJobs now support private images.
+* Gitops cluster tracers
+* Scheduled update checks
```
Comment (austinchambers) [inline content/release-notes/1.16.0.md:12]: At what interval are these checks performed by default? 

```suggestion
* Update checks within the Admin Console occur at periodic intervals and may be scheduled. 
```

File: content/release-notes/1.16.0.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+date: 2020-06-01T08:00:00-07:00
+linktitle: "1.16.0"
+title: "1.16.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+* Support for HTTP_PROXY and NO_PROXY environment variables.
+* CronJobs now support private images.
+* Gitops cluster tracers
```
Comment (austinchambers) [inline content/release-notes/1.16.0.md:11]: I don't understand these two above. The first seems customer-impacting (is it a bug?), the latter seems like it's technical and maybe can be removed?

File: content/release-notes/1.16.0.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+* Scheduled update checks
+* Support for custom redactors
```
Comment (austinchambers) [inline content/release-notes/1.16.0.md:13]:
```suggestion
* Support for [custom redactors](https://troubleshoot.sh/reference/redactors/overview/) 
```

File: content/release-notes/1.16.0.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+{{</features>}}
+
+{{<changes>}}
+* Velero storage credentials will now be validated on save.
```
Comment (austinchambers) [inline content/release-notes/1.16.0.md:17]:
```suggestion
* Snapshot storage configuration is now validated on save.
```

File: content/release-notes/1.16.0.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+* Updated troubleshoot dependency to v0.9.36
+* Helm 3 support
+* ConfigItem's `when` can now be a boolean
+* Namespaces will be applied before the rest of the application YAML.
+* Velero and Restic information is included in support bundles
```
Comment (austinchambers) [inline content/release-notes/1.16.0.md:22]:
```suggestion
* Additional snapshot information from Velero and Restic is now included in support bundles
```

File: content/release-notes/1.16.0.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+{{</changes>}}
+
+{{<fixes>}}
+* Fixed a bug that could cause `LocalImageName` and `LocalRegistryImagePullSecret` to panic.
```
Comment (austinchambers) [inline content/release-notes/1.16.0.md:26]: What happens when a panic occurs? Is this the following? 
```suggestion
* Fixed issue where Admin Console crashes when using `LocalImageName` and `LocalRegistryImagePullSecret`.
```

File: content/release-notes/1.16.0.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+* Snapshots page will show a message when connection to the API is lost.
```
Comment (austinchambers) [inline content/release-notes/1.16.0.md:27]: I understand the fix, but what happened when the connection was lost?

File: content/release-notes/1.16.0.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+* Fixed a bug that would cause orphaned pods to be left on the system when jobs are deleted.
+* Fixed a bug that could cause a mismatch between TLS cert and key when using `TLSCert` and `TLSKey` template functions
```
Comment (austinchambers) [inline content/release-notes/1.16.0.md:29]: I wouldn't call this a bug fix. Instead, I see this as a consistency improvement (that also breaks compatibility).

File: content/release-notes/1.16.0.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+* Added scrolling on airgap upload progress page.
```
Comment (austinchambers) [inline content/release-notes/1.16.0.md:30]:
```suggestion
* Airgap upload progress may be cropped without the ability to scroll in some situations. Fixed by adding the ability to scroll on this page.
```

File: content/release-notes/1.16.0.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+* Fixed a bug that allowed installing application with invalid YAML.
```
Comment (austinchambers) [inline content/release-notes/1.16.0.md:31]: What happened when you installed with invalid YAML?

File: content/release-notes/1.16.0.md (hunk 5 above)
Comment (marccampbell) [reply content/release-notes/1.16.0.md:13]: This is not a 1.16 feature, @laverya this was released in 1.15 IIRC?

File: content/release-notes/1.16.0.md (hunk 4 above)
Comment (marccampbell) [reply content/release-notes/1.16.0.md:11]: The "gitops cluster tracers" should be included as a change, but explained better...  @salahalsaleh will you take a pass at explaining the change here (... added annotations to ...)

File: content/release-notes/1.16.0.md (hunk 12 above)
Comment (emosbaugh) [reply content/release-notes/1.16.0.md:31]: How about `Warnings will now be visible in the browser admin console when installing an application with invalid YAML`?

Comment (emosbaugh) [inline content/release-notes/1.16.0.md:31]:
```suggestion
* Warnings will now be visible on the Version History page in the Admin Console when installing an application with invalid YAML
```

File: content/release-notes/1.16.0.md (hunk 4 above)
Comment (salahalsaleh) [reply content/release-notes/1.16.0.md:11]:
```suggestion
* Gitops cluster tracers - added the app slug and sequence as annotations to the application pods
```

File: content/release-notes/1.16.0.md (hunk 3 above)
Comment (salahalsaleh) [reply content/release-notes/1.16.0.md:12]: How about:

```suggestion
* Update checks within the Admin Console occur at periodic intervals (every 4 hours by default), and can be configured.
```

File: content/release-notes/1.16.0.md (hunk 10 above)
Comment (divolgin) [reply content/release-notes/1.16.0.md:29]: it's a backwards compatible change

File: content/release-notes/1.16.0.md (hunk 12 above)
Comment (emosbaugh) [reply content/release-notes/1.16.0.md:31]: We should remove this. There is no change for this release

File: content/release-notes/1.16.0.md (hunk 2 above)
Comment (areed) [reply content/release-notes/1.16.0.md:9]: You can add these to the kotsadm and kotsadm-api deployments. Kurl will add them there automatically if it's configured with a proxy.

File: content/release-notes/1.16.0.md (hunk 4 above)
Comment (marccampbell) [reply content/release-notes/1.16.0.md]: @salahalsaleh how about:

* Added KOTS metadata to all application manifests. A unique identifier for the local application and the version sequence will be injected as annotations to all deployed manifests.

File: content/release-notes/1.16.0.md (hunk 5 above)
Comment (marccampbell) [reply content/release-notes/1.16.0.md]: @divolgin let's remove this bullet

**Pull request merged**